build.gradle

```diff
@@ -200,7 +201,60 @@ def determineCommitId() {
   }
 }
 
-def spotlessApplyModules = ['']
+def excludedSpotlessModules = [':clients',
```

Comment:
noticed that #16198 will add a new module. Maybe you can include that in this PR even though #16198 is not merged yet?